test/unittest/CMakeLists.txt

```
@@ -103,6 +103,16 @@ if(is_dpcpp)
   )
 endif()
 
+if( (DEFINED TUNING_TARGET AND (${TUNING_TARGET} STREQUAL "INTEL_GPU")) OR
```

This would fail to be parsed if `TUNING_TARGET` is not defined despite the first condition. The quotes are the best practice to avoid this issue, that way you still get an empty string if `TUNING_TARGET` is not defined. (similarly below)
```suggestion
if( ("${TUNING_TARGET}" STREQUAL "INTEL_GPU") OR
```

---

Thank you for your comment @Rbiessy 
I was not aware of it. Updated in  [8116dbf](https://github.com/codeplaysoftware/portBLAS/pull/524/commits/8116dbf4453f9eaa3a51eaf1035355a06a165bfd)